### PR TITLE
Fix a minor typo

### PR DESCRIPTION
**Pull request by diokey**
Renamed removeRequster to removeRequester

### CODE REVIEW

File: modules/collectionRegistrationMixin.js [hunk 1]
```diff
@@ -213,7 +213,7 @@
        * Removes a requester from this cache. No longer receives updates
```
Comment (kentmw) [inline modules/collectionRegistrationMixin.js]: update line 212  and 160 to reflect this change.

Also run gulp bundle to update the bundle

**Comment by kentmw** [comment]
Thanks for catching this!

File: torso-bundle.js [hunk 2]
```diff
@@ -305,11 +305,56 @@
           },
 
           /**
+           * Will force the cache to fetch a subset of this collection's tracked ids
+           * @method requesterMixin.fetchSubsetOfTrackedIds
```
Comment (diokey) [inline torso-bundle.js]: I am not sure why this changed.

Comment (kentmw) [inline torso-bundle.js]: did you pull --rebase?

If you have, then it's possible the person before you didn't gulp bundle before merging

Comment (diokey) [inline torso-bundle.js]: I didn't git pull --rebase, but I was working on a fresh copy from master/upstream.

Comment (kentmw) [inline torso-bundle.js]: Then this is fine, I imagine it was someone not running a bundle before merging

+2

**Pull request merged**